UI/templates/widget.html

```diff
@@ -25,6 +25,10 @@
                data-dojo-props="required:true,value:'[% template_name %]'"
                data-dojo-attach-point="templateName">
             <span value=""><span>[% text("--TEMPLATE--") %]</span></span>
+            <span value="print_journal_entry">
+              <span>[% text("JE transaction") %]</span></span>
```

Comment:
I'm in doubt: the menu under which the print button resides is called "Journal Entry". That menu item is located within the "General Journal" menu. I'm inclined to call it a "General Journal transaction"... But elsewhere I *think* we may be calling the same thing a G/L transaction.

Reply:
Ok. I figured it out. I'd like it to be called "G/L transaction".

Reply:
Done.